troubleshooting/troubleshooting.md

```diff
@@ -639,15 +641,56 @@ Issue with AppleRTC, quite a simple fix:
 
 * config.plist -> Kernel -> Quirks -> DisableRtcChecksum -> true
 
-**Note**: If you still have issues, you'll need to use [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases) and exclude ranges. See [here for more info](https://github.com/acidanthera/bugtracker/issues/788#issuecomment-604608329)
+**Note**: If you still have issues, you'll need to use [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases) and exclude ranges.
 
-The following boot-arg should handle 99% of cases(pair this with RTCMemoryFixup):
+For a more in-depth guide, see here:
+
+* [Fixing RTC/CMOS Resets](https://dortania.github.io/OpenCore-Post-Install/misc/rtc.html)
+
+## Synaptics PS2 based trackpad doesn't work
+
+You can try to use the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl).
+First, you have to open DeviceManager, and head to the following:
+
+```
+Device Manager -> Mice and other pointing devices -> Properties -> Details > BIOS device name
+```
+
+Then grab the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl)
+By default, this uses PCI0.LPCB.PS2K for the pathing. you'll want to rename accordingly.
+
+```
+External (_SB_.PCI0.LPCB.PS2K, DeviceObj) <- Rename this
+
+    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package()  <- Rename this
+
+```
+
+* Note: Although this will work for some case, [the trackpad may be laggy and you can't use the physical buttons](https://github.com/acidanthera/bugtracker/issues/890). If you can't live without the trackpad, this may be better:
```

Comment:
```suggestion
* Note: Although this will work for most cases, the trackpad may be laggy and you may not be able to use the physical buttons ([more details](https://github.com/acidanthera/bugtracker/issues/890)). If you can live without the trackpad, this may be better:
```

Can or can't? From looking at the SSDT it appears to me to be can, but that makes no sense if you are trying to fix your touchpad...

Reply:
Sorry for my typo, it must be can instead of can't and if you want to look at my pull request, it is [here](https://github.com/dortania/OpenCore-Install-Guide/pull/122)